src/query.rs

```diff
@@ -75,7 +75,7 @@ impl SemverQuery {
     pub(crate) fn all_queries() -> BTreeMap<String, SemverQuery> {
         let mut queries = BTreeMap::default();
         for query_text in get_query_text_contents() {
-            let query: SemverQuery = ron::from_str(&query_text).unwrap_or_else(|e| {
+            let query: SemverQuery = ron::from_str(query_text).unwrap_or_else(|e| {
```

Comment:
Shouldn't that have been caught by clippy in my PR?

Reply:
`query_text` changed from `String` to `&'static str` here. The original code linted clean, but with the type change this was a needless borrow.